utils/openwrt-exporter/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2013-2017 OpenWrt.org
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwrt-exporter
```

Comment:
I suggest we change the name, as this is not limited to OpenWrt and also targets and works on LEDE. For example, **metrics-exporter** or even more precise **prometheus-exporter** (although other time series databases may in the future use the very same format)